test failures in Fedora rawhide with python3.9

This is with python3-3.9.0~b1-4.fc33.x86_64, python3-astroid-2.4.1-2.gita672051.fc33.noarch and asttokens=2.0.4:

```
=================================================== FAILURES ====================================================
___________________________________________ TestAstroid.test_fixture9 ___________________________________________
/usr/lib/python3.9/site-packages/astroid/builder.py:168: in _data_build
    node, parser_module = _parse_string(data, type_comments=True)
/usr/lib/python3.9/site-packages/astroid/builder.py:445: in _parse_string
    parsed = parser_module.parse(data + "\n", type_comments=type_comments)
/usr/lib/python3.9/site-packages/astroid/_ast.py:48: in parse
    return parse_func(string)
/usr/lib64/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "<unknown>", line 2
E       (*args)
E              ^
E   SyntaxError: invalid syntax

The above exception was the direct cause of the following exception:
tests/test_mark_tokens.py:172: in test_fixture9
    def test_fixture9(self): self.verify_fixture_file('astroid/module2.py')
tests/test_mark_tokens.py:134: in verify_fixture_file
    tested_nodes = m.verify_all_nodes(self)
tests/tools.py:96: in verify_all_nodes
    rebuilt_node = test_case.parse_snippet(text, node)
tests/test_mark_tokens.py:735: in parse_snippet
    return self.module.parse('_\n(' + text + ')').body[1].value
/usr/lib/python3.9/site-packages/astroid/builder.py:279: in parse
    return builder.string_build(code, modname=module_name, path=path)
/usr/lib/python3.9/site-packages/astroid/builder.py:142: in string_build
    module = self._data_build(data, modname, path)
/usr/lib/python3.9/site-packages/astroid/builder.py:170: in _data_build
    raise exceptions.AstroidSyntaxError(
E   astroid.exceptions.AstroidSyntaxError: Parsing Python code failed:
E   invalid syntax (<unknown>, line 2)
____________________________________________ TestAstroid.test_splat _____________________________________________
/usr/lib/python3.9/site-packages/astroid/builder.py:168: in _data_build
    node, parser_module = _parse_string(data, type_comments=True)
/usr/lib/python3.9/site-packages/astroid/builder.py:445: in _parse_string
    parsed = parser_module.parse(data + "\n", type_comments=type_comments)
/usr/lib/python3.9/site-packages/astroid/_ast.py:48: in parse
    return parse_func(string)
/usr/lib64/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "<unknown>", line 2
E       (*arr)
E             ^
E   SyntaxError: invalid syntax

The above exception was the direct cause of the following exception:
tests/test_mark_tokens.py:334: in test_splat
    m = self.create_mark_checker(source)
tests/test_mark_tokens.py:43: in create_mark_checker
    checker.verify_all_nodes(self)
tests/tools.py:96: in verify_all_nodes
    rebuilt_node = test_case.parse_snippet(text, node)
tests/test_mark_tokens.py:735: in parse_snippet
    return self.module.parse('_\n(' + text + ')').body[1].value
/usr/lib/python3.9/site-packages/astroid/builder.py:279: in parse
    return builder.string_build(code, modname=module_name, path=path)
/usr/lib/python3.9/site-packages/astroid/builder.py:142: in string_build
    module = self._data_build(data, modname, path)
/usr/lib/python3.9/site-packages/astroid/builder.py:170: in _data_build
    raise exceptions.AstroidSyntaxError(
E   astroid.exceptions.AstroidSyntaxError: Parsing Python code failed:
E   invalid syntax (<unknown>, line 2)
_________________________________________ TestAstroid.test_sys_modules __________________________________________
/usr/lib/python3.9/site-packages/astroid/builder.py:168: in _data_build
    node, parser_module = _parse_string(data, type_comments=True)
/usr/lib/python3.9/site-packages/astroid/builder.py:445: in _parse_string
    parsed = parser_module.parse(data + "\n", type_comments=type_comments)
/usr/lib/python3.9/site-packages/astroid/_ast.py:48: in parse
    return parse_func(string)
/usr/lib64/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "<unknown>", line 2
E       (*args)
E              ^
E   SyntaxError: invalid syntax

The above exception was the direct cause of the following exception:
tests/test_mark_tokens.py:639: in test_sys_modules
    self.create_mark_checker(source)
tests/test_mark_tokens.py:43: in create_mark_checker
    checker.verify_all_nodes(self)
tests/tools.py:96: in verify_all_nodes
    rebuilt_node = test_case.parse_snippet(text, node)
tests/test_mark_tokens.py:735: in parse_snippet
    return self.module.parse('_\n(' + text + ')').body[1].value
/usr/lib/python3.9/site-packages/astroid/builder.py:279: in parse
    return builder.string_build(code, modname=module_name, path=path)
/usr/lib/python3.9/site-packages/astroid/builder.py:142: in string_build
    module = self._data_build(data, modname, path)
/usr/lib/python3.9/site-packages/astroid/builder.py:170: in _data_build
    raise exceptions.AstroidSyntaxError(
E   astroid.exceptions.AstroidSyntaxError: Parsing Python code failed:
E   invalid syntax (<unknown>, line 2)
--------------------------------------------- Captured stdout call ----------------------------------------------
/usr/lib64/python3.9/importlib/_bootstrap.py
_________________________________________ TestMarkTokens.test_fixture9 __________________________________________
tests/test_mark_tokens.py:172: in test_fixture9
    def test_fixture9(self): self.verify_fixture_file('astroid/module2.py')
tests/test_mark_tokens.py:134: in verify_fixture_file
    tested_nodes = m.verify_all_nodes(self)
tests/tools.py:96: in verify_all_nodes
    rebuilt_node = test_case.parse_snippet(text, node)
tests/test_mark_tokens.py:735: in parse_snippet
    return self.module.parse('_\n(' + text + ')').body[1].value
/usr/lib64/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "<unknown>", line 2
E       (*args)
E              ^
E   SyntaxError: invalid syntax
___________________________________________ TestMarkTokens.test_splat ___________________________________________
tests/test_mark_tokens.py:334: in test_splat
    m = self.create_mark_checker(source)
tests/test_mark_tokens.py:43: in create_mark_checker
    checker.verify_all_nodes(self)
tests/tools.py:96: in verify_all_nodes
    rebuilt_node = test_case.parse_snippet(text, node)
tests/test_mark_tokens.py:735: in parse_snippet
    return self.module.parse('_\n(' + text + ')').body[1].value
/usr/lib64/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "<unknown>", line 2
E       (*arr)
E             ^
E   SyntaxError: invalid syntax
________________________________________ TestMarkTokens.test_sys_modules ________________________________________
tests/test_mark_tokens.py:639: in test_sys_modules
    self.create_mark_checker(source)
tests/test_mark_tokens.py:43: in create_mark_checker
    checker.verify_all_nodes(self)
tests/tools.py:96: in verify_all_nodes
    rebuilt_node = test_case.parse_snippet(text, node)
tests/test_mark_tokens.py:735: in parse_snippet
    return self.module.parse('_\n(' + text + ')').body[1].value
/usr/lib64/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "<unknown>", line 2
E       (*args)
E              ^
E   SyntaxError: invalid syntax
--------------------------------------------- Captured stdout call ----------------------------------------------
/usr/lib64/python3.9/importlib/_bootstrap.py
============================================ short test summary info ============================================
FAILED tests/test_astroid.py::TestAstroid::test_fixture9 - astroid.exceptions.AstroidSyntaxError: Parsing Pyth...
FAILED tests/test_astroid.py::TestAstroid::test_splat - astroid.exceptions.AstroidSyntaxError: Parsing Python ...
FAILED tests/test_astroid.py::TestAstroid::test_sys_modules - astroid.exceptions.AstroidSyntaxError: Parsing P...
FAILED tests/test_mark_tokens.py::TestMarkTokens::test_fixture9 -   File "<unknown>", line 2
FAILED tests/test_mark_tokens.py::TestMarkTokens::test_splat -   File "<unknown>", line 2
FAILED tests/test_mark_tokens.py::TestMarkTokens::test_sys_modules -   File "<unknown>", line 2
============================== 6 failed, 100 passed, 1 skipped, 1 warning in 6.34s ==============================
```

And it seems to all stem from trying to parse this snippet:
```
(Pdb) p data
'_\n(*args)'
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,4 @@
-dist: xenial
+dist: bionic
 language: python
 sudo: false
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,7 +9,7 @@ python:
   - 3.7
   - 3.8-dev
   - 3.9-dev
-  - pypy2.7-6.0
+  - pypy2.7
   - pypy3.5
 
 env:

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,7 +9,7 @@ python:
   - 3.7
   - 3.8-dev
   - 3.9-dev
-  - pypy2.7
+  - pypy2
   - pypy3.5
 
 env:

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,7 +10,7 @@ python:
   - 3.8-dev
   - 3.9-dev
   - pypy2
-  - pypy3.5
+  - pypy3
 
 env:
   global:

--- a/asttokens/util.py
+++ b/asttokens/util.py
@@ -143,7 +143,7 @@ def is_joined_str(node):
 
 
 def is_starred(node):
-  """Returns whether node is an expression node."""
+  """Returns whether node is a starred expression node."""
   return node.__class__.__name__ == 'Starred'
 
 

--- a/asttokens/util.py
+++ b/asttokens/util.py
@@ -142,6 +142,11 @@ def is_joined_str(node):
   return node.__class__.__name__ == 'JoinedStr'
 
 
+def is_starred(node):
+  """Returns whether node is an expression node."""
+  return node.__class__.__name__ == 'Starred'
+
+
 def is_slice(node):
   """Returns whether node represents a slice, e.g. `1:2` in `x[1:2]`"""
   # Before 3.9, a tuple containing a slice is an ExtSlice,

--- a/setup.cfg
+++ b/setup.cfg
@@ -26,6 +26,7 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
     Programming Language :: Python :: Implementation :: CPython
     Programming Language :: Python :: Implementation :: PyPy
 

--- a/tests/test_mark_tokens.py
+++ b/tests/test_mark_tokens.py
@@ -728,9 +728,12 @@ partial_sums = [total := total + v for v in values]
     # For expressions and statements, we add a dummy statement '_' before it because if it's just a
     # string contained in an astroid.Const or astroid.Expr it will end up in the doc attribute and be
     # a pain to extract for comparison
+    # For starred expressions, e.g. `*args`, we wrap it in a function call to make it parsable.
     indented = re.match(r'^[ \t]+\S', text)
     if indented:
       return self.module.parse('def dummy():\n' + text).body[0].body[0]
+    if util.is_starred(node):
+      return self.module.parse('f(' + text + ')').body[0].value.args[0]
     if util.is_expr(node):
       return self.module.parse('_\n(' + text + ')').body[1].value
     if util.is_module(node):

